doc/source/whatsnew/v0.22.0.txt

```diff
@@ -354,3 +354,4 @@ Other
 
 - Improved error message when attempting to use a Python keyword as an identifier in a ``numexpr`` backed query (:issue:`18221`)
 - Bug in :class:`Timestamp` where comparison with an array of ``Timestamp`` objects would result in a ``RecursionError`` (:issue:`15183`)
+- Bug in :func:`Series.__floordiv__`  and :func:`Series.__rfloordiv__` where operating on a scalar ``timedelta`` raises an exception (:issue:`18824`)
```

Comment:
don't put things in Other. This is conversion. Also note that a floordiv of scalar / Series was incorrect before.

Reply:
verify that we have a doc example in timedelta.rst as well.

Reply:
> Also note that a floordiv of scalar / Series was incorrect before.
 
Something other than the mention of `Series.__rfloordiv__`?